### PR TITLE
Add Change/Revoke RA Role Functionality

### PR DESCRIPTION
**Pull request by DRvanR**
- [x] revert middleware client bundle to dev-develop.

Depends on https://github.com/SURFnet/Stepup-Middleware-clientbundle/pull/30

### CODE REVIEW

**Comment by DRvanR** [comment]
:boom: @rjkip

File: app/Resources/translations/messages.en_GB.xliff [hunk 1]
```diff
@@ -374,8 +374,71 @@
...
+      <trans-unit id="66b3d0fa2ccade8a1fedf425c1300623ce937ff9" resname="ra.management.change_ra_role.modal.are_you_sure">
+        <jms:reference-file line="32">views/RaManagement/changeRaRole.html.twig</jms:reference-file>
+        <jms:reference-file line="40">views/RaManagement/changeRaRole.html.twig</jms:reference-file>
+        <source>ra.management.change_ra_role.modal.are_you_sure</source>
+        <target>Are you sure you want to give the Registration Authority listed below the selected role?</target>
+      </trans-unit>
+      <trans-unit id="3773fbc17a37a56eef350fbe58c5c40a3ceb2351" resname="ra.management.change_ra_role.modal.cancel">
+        <jms:reference-file line="61">views/RaManagement/changeRaRole.html.twig</jms:reference-file>
+        <source>ra.management.change_ra_role.modal.cancel</source>
+        <target>cancel</target>
```
Comment (rjkip) [inline app/Resources/translations/messages.en_GB.xliff]: Lowercase?

File: src/Surfnet/StepupRa/RaBundle/Controller/RaManagementController.php [hunk 2]
```diff
@@ -221,6 +231,100 @@ public function amendRaInformationAction(Request $request, $identityId)
...
+            $logger->warning(sprintf("RA listing for identity ID '%s' not found", $identityId));
+            throw new NotFoundHttpException(sprintf("RA listing for identity ID '%s' not found", $identityId));
+        }
+
+        $command = new RetractRegistrationAuthorityCommand();
+        $command->identityId = $identityId;
+
+        $form = $this->createForm('ra_management_retract_registration_authority', $command)->handleRequest($request);
+        if ($form->isValid()) {
+            if ($form->get('cancel')->isClicked()) {
```
Comment (rjkip) [inline src/Surfnet/StepupRa/RaBundle/Controller/RaManagementController.php]: :open_mouth:

**Comment by rjkip** [comment]
:rocket:

**Pull request merged**